docs/dev-guide/tscript/debugging.mdx

```diff
@@ -239,11 +246,19 @@ require('esbuild').buildSync({
 });
 ```
 
-See esbuild docs: https://esbuild.github.io/api/#keep-names
-
+See [esbuild docs](https://esbuild.github.io/api/#keep-names) for more information: https://esbuild.github.io/api/#keep-names
 </TabItem>
 </Tabs>
 
+Another option to prevent this bundle issue is to overwrite the function name with `Object.defineProperty`:
```

Comment:
I would remove this part, see @mjameswh 's comment in the Jira ticket

Reply:
Done. Thank you.